spec/cypress/integration/CipStoriesAdmin.feature

```diff
@@ -84,3 +84,26 @@ Feature: Admin user interaction with Core Induction Programme
     Then "page body" should contain "Your changes have been saved"
     And "page body" should contain "Lesson part test title"
     And "page body" should contain "New content for lesson part"
+
+  Scenario: Can edit mentor materials
```

Comment:
I'm not really a fan of this being in this file, it feels like a bunch of basically unrelated features are ending up in this file purely because of who they're available to - but not sure if splitting it up is a task for this PR or another

Reply:
Ehh, I think I prefer that to be another PR - cause it's not just this file. It's also `CIPStoriesECT` :/